frontend/src/components/Banner.module.scss

```diff
@@ -79,6 +79,10 @@
 
   .icon {
     margin-right: $spacing-md;
+
+    .warning & {
+      color: $color-yellow-50;
+    }
```

Comment:
That's because rather than having different assets in different colours in our repository, I've been trying to use [`currentColor`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword) in SVGs, so that we can just use the right colour in the component we're using, while re-using the same single asset.